src/transforms/group.js

```diff
@@ -34,31 +34,27 @@ import {
 } from "../options.js";
 import {basic} from "./basic.js";
 
-/** @jsdoc groupZ */
+// Group on {z, fill, stroke}.
```

Comment:
should these use jsdoc blocks? 
```suggestion
/** Group on {z, fill, stroke}. */
```

Reply:
This is an internal-only comment, so I think `//` is more appropriate. Public comments live exclusively in the TypeScript declarations.

Reply:
Got it!